docs/4.0/utilities/display.md

```diff
@@ -70,13 +68,26 @@ To show an element only on a given interval of screen sizes you can combine one
 | Visible only on lg | `d-none d-lg-block d-xl-none` |
 | Visible only on xl | `d-none d-xl-block` |
 
+{% example html %}
+<div class="d-lg-none">hide on screens wider than lg</div>
+<div class="d-none d-lg-block">hide on screens smaller than lg</div>
+{% endexample %}
+
 ## Display in print
 
-Change the `display` value of elements when printing with our print display utilities.
+Change the `display` value of elements when printing with our print display utility classes.
 
 | Class | Print style |
 | --- | --- |
 | `.d-print-block` | Applies `display: block` to the element when printing |
 | `.d-print-inline` | Applies `display: inline` to the element when printing |
 | `.d-print-inline-block` | Applies `display: inline-block` to the element when printing |
 | `.d-print-none` | Applies `display: none` to the element when printing |
+
+The print and display classes can be combined.
+
+{% example html %}
+<div class="d-print-none">Screen Only (hide on print)</div>
+<div class="d-none d-print-block">Print Only (hide on screen)</div>
+<div class="d-none d-lg-block d-print-block">(lg or wider) screen and print only. Hide on smaller than lg screen.</div>
```

Comment:
Change text to `Hide up to large on screen, but always show on print`

Reply:
ok